.gitlab-ci.yml

```diff
@@ -128,6 +139,8 @@ publish:s3:apt-repo:
     - aws s3 sync dists s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/dists
     - aws s3 ls --recursive s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/dists/ |
       awk '{cmd="aws s3api put-object-acl --acl public-read --bucket ${S3_BUCKET_NAME} --key "$4; system(cmd)}'
+  after_script:
+    - aws s3 rm s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/lock
```

Comment:
Suggest putting an expiration timer on this file in S3, so it auto deletes itself if it becomes stale. Not sure if it is retained if the file is deleted and recreated though...

Reply:
I don't really know how these expiration works. I don't promise, but maybe I rework this later on.